flink-python/pyflink/table/table.py

```diff
@@ -1077,6 +1078,91 @@ def explain(self, *extra_details: ExplainDetail) -> str:
         j_extra_details = to_j_explain_detail_arr(extra_details)
         return self._j_table.explain(TEXT, j_extra_details)
 
+    def insert_into(
+        self, table_path_or_descriptor: Union[str, TableDescriptor], overwrite: bool = False
+    ) -> TablePipeline:
+        """
+        When ``target_path_or_descriptor`` is a table path:
+
+            Declares that the pipeline defined by the given :class:`Table` (backed by a
+            DynamicTableSink) should be written to a table that was registered under the specified
+            path.
+
+            See the documentation of
+            :func:`pyflink.table.table_environment.TableEnvironment.use_database` or
+            :func:`pyflink.table.table_environment.TableEnvironment.use_catalog` for the rules on
+            the path resolution.
+
+            Example:
+            ::
+
+                >>> table = table_env.sql_query("SELECTFROM MyTable")
```

Comment:
"SELECTFROM MyTable" does not look like valid SQL. I was expecting something like `SELECT * FROM MyTable; `